crates/youki/src/commands/info.rs

```diff
@@ -223,6 +224,38 @@ pub fn print_namespaces() {
     }
 }
 
+#[inline]
+fn get_cap_supported(caps: &caps::CapsHashSet, cap: caps::Capability) -> &'static str {
```

Comment:
can we rename this to `is_cap_supported` or`is_cap_available`  depending on how we are wording the output?